debug_gym/gym/envs/env.py

```diff
@@ -257,8 +257,24 @@ def _reset_env_state(self):
 
     def set_entrypoints(self, entrypoint: str, debug_entrypoint: str | None = None):
         if entrypoint:
-            self.entrypoint = entrypoint
-            debug_entrypoint = debug_entrypoint
+            is_python = "python " in entrypoint
+
+            self.entrypoint = self._prepare_entrypoint(entrypoint)
+
+            if debug_entrypoint is None:
+                debug_entrypoint = (
+                    entrypoint.replace("python ", "python -m pdb ", 1)
+                    if is_python else entrypoint
+                )
+
+            self.debug_entrypoint = self._prepare_entrypoint(debug_entrypoint)
+
+            if is_python and "-m pdb" not in self.debug_entrypoint:
```

Comment:
These should be outside `if entrypoint` right? And we might want to check that `self.debug_entrypoint is not None` too.